capstone/target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
```

Comment:
The entire target directory can probably be in .gitignore.

Reply:
done